app/views/plans/new.html.erb

```diff
@@ -56,7 +56,7 @@
                           form: org_fields,
                           id_field: :id,
                           default_org: nil,
-                          orgs: @orgs,
+                          orgs: @orgs.filter { |org| org.managed == true },
```

Comment:
Could we add this filter in the controller instead of the view ?

Reply:
Ah the view is `local_only`. If you check row 54 you can see it (not shown in the Github review window though). The `local_only` view is used multiple times around the app (for example, row 83 in this file also refers to it, as well as other files). If we modify the corresponding controller, all the places called `local_only` will be affected